src/main/java/com/shopify/ShopifySdk.java

```diff
@@ -117,6 +119,8 @@
 	static final String ACCESS_TOKEN = "access_token";
 	static final String PRODUCTS = "products";
 	static final String VARIANTS = "variants";
+	static final String CUSTOM_COLLECTIONS = "custom_collections";
+	static final String COLLECTS = "collects";
```

Comment:
I don't see that this is used anywhere. Please remove this.

Reply:
Ok. I'll reintroduce it in next PR for Collects support